.github/workflows/release-python.yml

```diff
@@ -0,0 +1,88 @@
+          CIBW_TEST_SKIP: '*universal2:arm64'
+          CIBW_BUILD: "cp36-macosx_x86_64 cp38-macosx_universal2 cp36-win* cp36-manylinux_{x86_64,i686}"
+      - uses: actions/upload-artifact@v2
+        with:
+          name: python_bsonjs-3.6-${{ matrix.os }}-wheel
+          path: ./wheelhouse/*.whl
+          if-no-files-found: error
+  test-non-linux-wheels:
+    needs: build-wheels
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-2019]
+        python-version: [3.6, 3.7, 3.8, 3.9, "3.10"]
+    name: Test CPython ${{ matrix.python-version }}-${{ matrix.os }}
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Download a previously created wheel
+        uses: actions/download-artifact@v2
+        with:
+          name: python_bsonjs-3.6-${{ matrix.os }}-wheel
+      - name: Test wheel
+        shell: bash
+        run: |
+          python -m pip install -U pip
+          python -m pip install --no-index --find-links=./ python_bsonjs
+          python -m pip list | grep python-bsonjs
+          python -c "from bsonjs import dumps"
+  # Linux
+  test-manylinux-wheels:
+    runs-on: ${{ matrix.os }}
+    needs: build-wheels
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        container: ['manylinux2014_i686', 'manylinux2014_x86_64']
+        python-version: ['cp36-cp36m', 'cp37-cp37m', 'cp38-cp38',
+                         'cp39-cp39', 'cp310-cp310']
+    name: Test CPython ${{ matrix.python-version }}-${{ matrix.container }}
+    steps:
+      - name: Download a previously created wheel
+        uses: actions/download-artifact@v2
+        with:
+          name: python_bsonjs-3.6-${{ matrix.os }}-wheel
+      - name: Test wheel
+        run: |
+          docker run --rm --volume `pwd`:/python quay.io/pypa/${{ matrix.container }} /bin/bash -c "/opt/python/${{ matrix.python-version }}/bin/python -m pip install -U pip && /opt/python/${{ matrix.python-version }}/bin/python -m pip install --find-links=/python/ --no-index python_bsonjs &&  /opt/python/${{ matrix.python-version }}/bin/python -m pip list | grep python-bsonjs && /opt/python/${{ matrix.python-version }}/bin/python -c 'from bsonjs import dumps'"
+  make_sdist:
+    name: Make SDist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build SDist
+        run: python setup.py sdist --dist-dir=dist --formats=zip
+      - uses: actions/upload-artifact@v2
+        with:
+          name: "sdist"
+          path: dist/*.zip
+  collect-wheels:
+    runs-on: ubuntu-latest
+    needs: [build-wheels, make_sdist]
+    name: Download Wheels
+    steps:
+      - name: Download all workflow run artifacts
+        uses: actions/download-artifact@v2
+      - uses: actions/upload-artifact@v2
+        with:
+          name: all-wheels
+          path: "*"
```

Comment:
One last question. When I download and unpack the archive this is the structue:
```
$ tree all-wheels
all-wheels
├── python_bsonjs-3.6-macos-latest-wheel
│   ├── python_bsonjs-0.3.0.dev0-cp36-abi3-macosx_10_9_universal2.whl
│   └── python_bsonjs-0.3.0.dev0-cp36-abi3-macosx_10_9_x86_64.whl
├── python_bsonjs-3.6-ubuntu-latest-wheel
│   ├── python_bsonjs-0.3.0.dev0-cp36-abi3-manylinux_2_12_i686.manylinux2010_i686.manylinux_2_17_i686.manylinux2014_i686.whl
│   └── python_bsonjs-0.3.0.dev0-cp36-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
├── python_bsonjs-3.6-windows-2019-wheel
│   ├── python_bsonjs-0.3.0.dev0-cp36-abi3-win32.whl
│   └── python_bsonjs-0.3.0.dev0-cp36-abi3-win_amd64.whl
└── sdist
    └── python-bsonjs-0.3.0.dev0.zip
```

Is that intentional? I would expect all the files in one directory.

Reply:
Fixed.